### PR TITLE
Feature/#1549 genesis txs

### LINKED ISSUE #1549: Add genesis transactions
Include genesis txs into the transactions table in db.
Test e2e to get those txs via rpc

### PR DESCRIPTION
**Pull request by ARR552**
Closes #1549 

### What does this PR do?

This pr adds the genesis txs and receipts 

### Reviewers

- @arnaubennassar 
- @ToniRamirezM

### CODE REVIEW

File: test/Makefile [hunk 1]
```diff
@@ -212,9 +212,6 @@ run-l2-explorer: ## Runs L2 blockscan explorer
 	$(RUNEXPLORERJSONRPC)
 	$(RUNEXPLORERL2)
 
-.PHONY: run-explorer
-run-explorer: run-explorer-l1 run-explorer-l2 ## Runs both L1 and L2 explorers
-
```
Comment (tclemos) [inline test/Makefile:217]: Why did you remove this? please keep it

File: test/docker-compose.yml [hunk 2]
```diff
@@ -256,6 +256,7 @@ services:
       - ZKEVM_NODE_RPC_PORT=8124
       - ZKEVM_NODE_RPC_WEBSOCKETS_PORT=8134
     volumes:
+      - ./test.keystore:/pk/keystore
```
Comment (tclemos) [inline test/docker-compose.yml:259]: jRPC shouldn't need a private key, why did you add this?

**Review by tclemos — CHANGES_REQUESTED** [review]

File: test/Makefile (hunk 1 above)
Comment (ARR552) [reply test/Makefile:217]: because this command was not working. `run-explorer-l1` was removed in another PR. Also, to run `run-explorer-l2` we have run-l2-explorer

**Review by arnaubennassar — APPROVED** [review]
